spec/Classes.tex

```diff
@@ -838,6 +893,28 @@ \subsection{Class Assignment}
 variable of a class type to another, both variables reference the same
 class instance.
 
+Assignment for \chpl{owned} variable transfers ownership and leaves the
+source variable empty. For example:
+
+\begin{chapelexample}{owned-assignment.chpl}
+Supposing that \chpl{C} is a class type:
+\begin{chapelpre}
+class C { }
+\end{chapelpre}
+\begin{chapel}
+var a = new owned C();
```

Comment:
Instead of "Supposing that \chpl{C} is a class type" above, I suggest to add a comment in the `var a .....` line above saying, ex., `// assume C is a class` (or ...... class type). This should make it more organic / connected to the code.